.travis.yml

```diff
@@ -29,8 +29,10 @@ node_js:
 
 ## Use this to prepare your build for testing
 ## e.g. copy database configurations, environment variables, etc.
-# before_script:
-#
+## http://stackoverflow.com/questions/12336566/travis-ci-with-jasmine-node
+before_script:
```

Comment:
The problem is our scripts for ``npm install`` need the server to run the tests against. So we need to start it before the scripts.

But I just tried it, and it looks like running the server halts all further travis operations, and stalls until it times out. So it looks like we'll need to find some way to run the server in the background of the terminal.